internal/observability/graph.go

```diff
@@ -36,8 +36,7 @@ import (
 var (
 	_timeNow          = time.Now // for tests
 	_defaultGraphSize = 128
-	// Latency buckets for histograms. At some point, we may want to make these
-	// configurable.
+	// Default latency buckets for histograms.
```

Comment:
Did we loose the changes made in [#2104](https://github.com/yarpc/yarpc-go/pull/2104)? or you plan to extend the buckets through service configuration?